robot_assignments/web/JetAutomateExercise/testcases/testDoppio.robot

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+Resource    ${CURDIR}/../import.robot
+
+
+*** Variable ***
+${urlLogin}    http://125.26.15.143:13132/ 
+${username}     oz4899
+${password}     1234
+${username_wrong}   abcd
+${searchText}   shirt
+${searchText_wrong}   abcd
+
+*** Test Cases ***
+Testcase 01 - Verify that user cannot login with wrong username
+    Open Website Doppio
+    Login With Username And Password  ${username_wrong}    ${password}
+    Verify Login failed
+    Close Browser
+
+Testcase 02 - Verify that user can login to the system when username are correct
+    Open Website Doppio
+    Login With Username And Password   ${username}    ${password}
+    Verify Login Pass
+    Close Browser
+
+Testcase03 - Verify that search function are working correctly
+    Open Website Doppio
+    Login With Username And Password   ${username}    ${password}
+
+    Wait Until Element Is Visible       ${searchFieldXpath} 
+    Input Text To Search Bar      ${searchText_wrong}
+    Click Search 
+    Wait Until Page Does Not Contain Element        ${itemCardXpath}
+    Wait Until Element Is Visible       ${notFoundXpath}
+    Verify No Result Found
+
+    Input Text To Search Bar     ${searchText}
+    Click Search   
+    Wait Until Element Is Visible   ${itemListXpath}
+    Verify That Search Result Contain Search Text  ${searchText}
```

Comment:
feature: Verify That Search Result Contain Search Text  ${searchText}
steps:
Wait Until Element Is Visible   ${itemListXpath}
ตามด้วย step ที่เราเขียนก็ได้
    ${elements}=    Get WebElements    ${itemListXpath} 
    FOR     ${element}   IN     @{elements}
        ${elementText}  Get Text   ${element}
        ${lower_case_text}=    Convert To Lower Case     ${elementText}
        Should Contain      ${lower_case_text}    ${searchText}
    END